spec/models/oauth_client_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe OauthClient, type: :model do
+  def create_oauth_client(client_id, client_name, redirect_uri)
```

Comment:
You don't need to change this now but as you add more files & tests I'd look into using [`factory_bot_rails`](https://github.com/thoughtbot/factory_bot_rails) to manage your classes during tests.